### PR TITLE
Notes from Core WG 04/17/2020 call

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

Notes from Core WG call  04/17/2020.

### Testing Strategy

N/A

### TODO or Help Wanted

None

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make formatall`.

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2020-04-17.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * Brad Campbell
+ * Vadim Sukhomlinov
+ * Johnathan Van Why
+ * Pat Pannuto
+ * Samuel Jero
+ * Alistair
+ * Garret Kelly
+ * Philip Levis
+ * Andrey Pronin
+ * Hudson Ayers
```
Comment (lschuermann) [inline doc/wg/core/notes/core-notes-2020-04-17.md:15]: I did join fairly late; it might create confusion to see my comments while not having attended. :)

Comment (brghena) [reply doc/wg/core/notes/core-notes-2020-04-17.md:15]: Fixed!  Sorry about that.

Comment (lschuermann) [reply doc/wg/core/notes/core-notes-2020-04-17.md:15]: Thanks and no worries!

**Pull request merged**